### PR TITLE
Comments: Fix comment detail preview line-clamp fallback gradient

### PR DESCRIPTION
**Pull request by Copons**
Fixes https://github.com/Automattic/wp-calypso/issues/15590

In browsers not supporting `line-clamp` and using the gradient workaround:
- Add a yellow gradient for pending comments, to match the yellow background.
- Remove the gradient on placeholders.

screen shot 2017-06-28 at 14 02 34

### CODE REVIEW

File: client/blocks/comment-detail/style.scss [hunk 1]
```diff
@@ -20,7 +20,8 @@
 	// If the comment is unapproved and collapsed, color it yellow.
 	&.is-unapproved.is-collapsed {
 		.comment-detail__header {
-			background: rgba( $alert-yellow, 0.085 );
+			//background: rgba( $alert-yellow, 0.085 );
```
Comment (gwwar) [inline client/blocks/comment-detail/style.scss]: Don't forget to remove this

File: client/blocks/comment-detail/style.scss [hunk 2]
```diff
@@ -20,7 +20,8 @@
...
+			background: mix( $alert-yellow, $white, 8.5% );
```
Comment (gwwar) [inline client/blocks/comment-detail/style.scss]: I had to look this one up :) http://sass-lang.com/documentation/Sass/Script/Functions.html#mix-instance_method

**Comment by drw158** [comment]
Is there an easy way to test this?

**Comment by gwwar** [comment]
Tested well on my Desktop 👍

File: client/blocks/comment-detail/style.scss (hunk 1 above)
Comment (Copons) [inline client/blocks/comment-detail/style.scss]: I think I removed this but I haven't pushed it :|

**Comment by Copons** [comment]
@drw158 I'm about to ship this anyway, but just FYI line-clamp is unsupported on Firefox too. :)

**Pull request merged**